code.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27130.2027
+# Visual Studio Version 16

```

Comment:
why this change ?

Reply:
I didn't make this change, it got pushed automatically in the code.sln file. Might be due to VS update maybe